micro-rdk/src/common/registry.rs

```diff
@@ -214,21 +232,21 @@ impl ComponentRegistry {
     #[cfg(feature = "camera")]
     pub fn register_camera(
         &mut self,
-        model: &'static str,
+        model: String,
```

Comment:
If this is written as:

```
pub fun register_camera<S: Into<String>>(&mut self, name: S)
```

Then you should be able to pass either `String` or `&str`, and existing callers will not need to change.

Reply:
implemented